drivers/dai/intel/dmic/dmic.c

```diff
@@ -852,7 +873,9 @@ static int dai_dmic_initialize_device(const struct device *dev)
 		dai_dmic_irq_handler,
 		DEVICE_DT_INST_GET(0),
 		0);
-	return 0;
+
+	pm_device_init_suspended(dev);
+	return pm_device_runtime_enable(dev);
```

Comment:
Just to make it clear. `pm_device_init_suspended` is only needed if the device is *at this moment physically suspended.
After `pm_device_runtime_enable` the device will be **suspended** and we will need to call probe to resume it.